### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by mpr0xy**
Fix typos

### CODE REVIEW

**Review by sbraveyoung — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -166,4 +166,4 @@ The decoder, encoder and filter reference [ffmpeg cmdline tool](http://ffmpeg.or
 
 The project also draws inspiration from other popular frameworks, such as [ffmpeg-python](https://github.com/kkroening/ffmpeg-python) and [mediapipe](https://github.com/google/mediapipe). Our [website](https://babitmf.github.io/) is using the project from [docsy](https://github.com/google/docsy) based on [hugo](https://github.com/gohugoio/hugo).
 
-Here, we'd like to express our sincerest thanks to the developers of the above projects!
\ No newline at end of file
+Here, we'd like to express our sincerest thanks to the developers of the above projects!
```
Comment (frankfengw519) [inline README.md:169]: It seems that there's no changes of this line？

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # BMF - Cross-platform, customizable video processing framework with strong GPU acceleration
 
 **BMF (Babit Multimedia Framework)** is a cross-platform, customizable multimedia processing framework developed by [**ByteDance**](https://www.bytedance.com/en).
-With over 4 years of testing and improvements, BMF has been tailored to adeptly tackle challenges in our real-world production environments. Now it's widely used in ByteDance's video streaming, live transcoding, cloud editing and mobile pre/post processing scenarios. More than 2 bilion videos are processed by the framework everyday.
+With over 4 years of testing and improvements, BMF has been tailored to adeptly tackle challenges in our real-world production environments. Now it's widely used in ByteDance's video streaming, live transcoding, cloud editing and mobile pre/post processing scenarios. More than 2 billion videos are processed by the framework everyday.
```
Comment (frankfengw519) [inline README.md:4]: Thanks very much！

File: README.md (hunk 1 above)
Comment (mpr0xy) [reply README.md:169]: Yes, I haven't made any changes to this line.

**Pull request merged**